reference/toolstacks/nodejs/README.md

```diff
@@ -20,59 +20,70 @@ To use Platform.sh and Node.js together, configure the ``.platform.app.yaml``
 file with a few key settings, as described here (a complete example is included
 at the end).
 
-1. Specify the `type` as `nodejs` you should definitely specify a version for example `type: nodejs:4.2`
+1. Specify the `type` as `nodejs`. You may also specify a version for example `type: nodejs:4.4`.
 2. Specify your dependencies under the `nodejs` key, like this:
 
-```yaml
- dependencies:
-   nodejs:
-     pm2: "^0.15.10"
-```
+   ```yaml
+   dependencies:
+     nodejs:
+       pm2: "^0.15.10"
+   ```
 
-These are the global dependencies of your project (the ones you would have
-installed with `npm install -g`). Here we specify the pm2, Production process manager that will allow us to run the node process.
+   These are the global dependencies of your project (the ones you would have
+   installed with `npm install -g`). Here we specify the `pm2`, process manager
+   that will allow us to run the node process.
 
 3. Configure the command you use to start serving your application (this must
    be a foreground-running process) under the `web` section, e.g.:
 
-```yaml
- web:
-   commands:
-     start: "PM2_HOME=/app/run pm2 start index.js --no-daemon"
-```
-
-If there is a package.json file present at the root of your repository 
-Platform.sh will automatically install the dependencies. We suggest including
-the `platformsh` helper npm module, that makes it trivial to access the 
-running environement.
-
-```javascript
-  "dependencies": {
-    "platformsh": "^0.0.1"
-  }
-```
+   ```yaml
+   web:
+     commands:
+       start: "PM2_HOME=/app/run pm2 start index.js --no-daemon"
+   ```
+
+   If there is a package.json file present at the root of your repository,
+   Platform.sh will automatically install the dependencies. We suggest including
+   the `platformsh` helper npm module, that makes it trivial to access the
+   running environement.
+
+   ```javascript
+   "dependencies": {
+     "platformsh": "^0.0.1"
+   }
+   ```
+
+4. Create any Read/Write mounts. The root file system is read only,
+you have to explicitly describe writable mounts. In (3) we set the
+home of the process manager to `/app/run` so this needs to be writable.
+
+   ```yaml
+   mounts:
+     "/run": "shared:files/run"
+   ```
```

Comment:
This is slightly off topic, but we mount /run to files/run because we specified /app/run... The point all of these paths are to is extremely confusing. This is not a nodejs-specific problem, but something we need to correct system-wide.